concrete/single_pages/dashboard/system/conversations/settings.php

```diff
@@ -14,8 +14,13 @@
         <legend><?php echo t('Attachment Settings'); ?></legend>
         <p class="help-block"><?php echo t('Note: These settings can be overridden in the block edit form for individual conversations.'); ?></p>
         <div class="form-group">
-            <label class="control-label"><?= t('Enable Attachments') ?></label>
-            <?= $form->checkbox('attachmentsEnabled', 1, $attachmentsEnabled) ?>
+            <label class="control-label"><?= t('Attachments') ?></label>
+            <div class="checkbox">
+                <label>
+                    <?= $form->checkbox('attachmentsEnabled', 1, $attachmentsEnabled) ?>
+                    <?= t('enable file attachments') ?>
```

Comment:
I think this should probably be capitalized

Reply:
Roger, roger. I will update this.